dashboard/test/controllers/registrations_controller_test.rb

```diff
@@ -300,23 +300,6 @@ class RegistrationsControllerTest < ActionController::TestCase
       assigns(:user).errors.full_messages
   end
 
-  test 'sign up as teacher' do
```

Comment:
We now adjust the display of age for a student, and the passing of an age for a teacher, in javascript, and my understanding is that these unit tests run against an html parser but not against any kind of javascript virtual machine.  Fortunately we still have UI tests that do sign-up for both student and teacher.